portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -52,11 +56,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get comments from the form
     String name = getParameter(request, /*name=*/"name", /*defaultValue=*/"");
```

Comment:
Maybe not too late to change the request param name to be the same as Comment attributes?

Reply:
Fixed